pkg/storage/etcd3/watch_test.go

```diff
@@ -0,0 +1,175 @@
+/*
```

Comment:
Since you renamed the non-test file to "watcher.go", how about renaming it to "watcher_test.go"?